src/main/java/pw/kaboom/commandspy/Main.java

```diff
@@ -66,10 +66,14 @@ public boolean onCommand(final CommandSender sender, final Command cmd, final St
 
 	@EventHandler
 	void onPlayerCommandPreprocess(final PlayerCommandPreprocessEvent event) {
+		if (event.isCancelled()) {
+			return;
+		}
+
 		for (String uuidString : config.getKeys(false)) {
 			UUID uuid = UUID.fromString(uuidString);
 
-			if (Bukkit.getPlayer(uuid) != null) {
+			if (Bukkit.getPlayer(uuid) != null && !event.isCancelled()) {
```

Comment:
```suggestion
			if (Bukkit.getPlayer(uuid) != null) {
```

Reply:
Are you moving the check?

Reply:
Because that code would remove the functionality of this pull request altogether

Reply:
Yes, this check is superfluous after the other one was added outside the loop.